ci-operator/step-registry/aws/provision/route53/aws-provision-route53-commands.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
```

Comment:
IMO, it's better that update the name to reflect the AWS resources we are trying to create, like `aws-provision-route53-private-hosted-zone` (the `private-hosted-zone` is a subfolder of `route53`)

Reply:
Done